### DISCUSSION TITLE
ARM64 support

### DISCUSSION THREAD

**Discussion by ibraheemalayan** [tiangolo/uwsgi-nginx-flask-docker#380] [Questions] [answered]
Are there any intentions to support arm64/aarch64 Architectures, if not what are possible alternatives

**Comment by jondurrant** [comment]
Support having an arm64 image too.

**Comment by chowyu12** [comment]
mac m1 start error：
`io_setup() failed (38: Function not implemented)`

**Comment by chowyu12** [comment]
+1

**Comment by tiangolo** [comment, marked as answer]
There's support for Arm for a while ago, I didn't post here then. :sweat_smile: :nerd_face:
